schemas/dex/SCHEMA.md

```diff
@@ -0,0 +1,186 @@
+|-----------------------|----------------------------------------------------------------------|--------|
+| timestamp             | The timestamp of the transaction.                                    | number |
+| chain_id              | The standard id of the chain.                                        | number |
+| block_number          | The block number of the trade.                                       | number |
+| log_index             | The log index of the event recorded.                                 | number |
+| transaction_hash      | The hash of the transaction.                                         | string |
+| user_address          | The address that initiates the transaction (ie, the transaction signer). | string |
+| contract_address      | The smart contract address the user interacted with (ie, transaction to). | string |
+| taker_address         | The taker, the address that receives the output of the swap (ie, could be the same as user_address unless a proxy contract/router is used). | string |
+| pool_address          | The contract address of the pool.                                    | string |
+| token0_address        | The contract address of token0.                                      | string |
+| token0_amount         | The amount of token0.                                                | number |
+| token1_address        | The contract address of token1.                                      | string |
+| token1_amount         | The amount of token1.                                                | number |
+
+### V2 Transfers
+| Property              | Description                                                          | Type   |
+|-----------------------|----------------------------------------------------------------------|--------|
+| timestamp             | The timestamp of the transaction.                                    | number |
+| chain_id              | The standard id of the chain.                                        | number |
+| block_number          | The block number of the trade.                                       | number |
+| log_index             | The log index of the event recorded.                                 | number |
+| transaction_hash      | The hash of the transaction.                                         | string |
+| user_address          | The address that initiates the transaction (ie, the transaction signer). | string |
+| contract_address      | The smart contract address the user interacted with (ie, transaction to). | string |
+| taker_address         | The taker, the address that receives the output of the swap (ie, could be the same as user_address unless a proxy contract/router is used). | string |
+| pool_address          | The contract address of the pool.                                    | string |
+| pool_token_balance    | The balance of the pool token, decimal normalized.                   | number |
```

Comment:
this schema seems wrong.

should be something like this :

timestamp
chain_id
block_number
log_index
transaction_hash
pool_address / contract_address (lp pool address)
from_address - from address that sends the lp token
to_address - to address that receives the lp token
pool_token_balance